packages/node-core/CHANGELOG.md

```diff
@@ -10,6 +10,15 @@ All logs must start with the format: [x.y.z] - yyyy-mm-dd
 
 ## [Unreleased]
 
+
+## [2.0.0] - 2023-04-20
+- Major release 2.0.0
+### Added
+- Added StoreCache service and various other improvements (#1561)
+- Added TestingService (#1584)
+### Fixed
+- Enable Typescript strict setting and improve code quality (#1625)
```

Comment:
- Fixed enum not being included in POI
- Was there some offest bug with poi/mmr too?
- Some DB index related fixes?
- Fixed dictionary validation issues and improve error messages